Add zone config troubleshooting guide (#19283)
cockroachdb/docs: src/current/_includes/v24.3/backups/zone-configs-overwritten-during-restore.md

> [Zone configurations]({% link {{ page.version.version }}/configure-replication-zones.md %}) present on the destination cluster prior to a restore will be **overwritten** during a [cluster restore]({% link {{ page.version.version }}/restore.md %}#full-cluster) with the zone configurations from the [backed up cluster]({% link {{ page.version.version }}/backup.md %}#back-up-a-cluster). If there were no customized zone configurations on the cluster when the backup was taken, then after the restore the destination cluster will use the zone configuration from the [`RANGE DEFAULT` configuration]({% link {{ page.version.version }}/configure-replication-zones.md %}#view-the-default-replication-zone).

Reviewer: Suggestions for readability 

```suggestion
During a [cluster restore]({% link {{ page.version.version }}/restore.md %}#full-cluster), any [zone configurations]({% link {{ page.version.version }}/configure-replication-zones.md %}) present on the destination cluster are **overwritten** with the zone configurations from the [backed-up cluster]({% link {{ page.version.version }}/backup.md %}#back-up-a-cluster). If no customized zone configurations were on the cluster when the backup was taken, then after the restore the destination cluster will use the zone configuration from the [`RANGE DEFAULT` configuration]({% link {{ page.version.version }}/configure-replication-zones.md %}#view-the-default-replication-zone).
```

Author: ✅ 🙇
